base/compiler/ssair/ir.jl

```diff
@@ -88,7 +88,7 @@ function compute_basic_blocks(stmts::Vector{Any})
     bb_starts = basic_blocks_starts(stmts)
     # Compute ranges
     pop!(bb_starts, 1)
-    basic_block_index = collect(bb_starts)
+    basic_block_index = sort!(collect(bb_starts); alg=QuickSort)
```

Comment:
Do we need to unique this also, or at least skip duplicated entries in the processing below?

Reply:
It should already be unique, as we `collect` it from `BitSet`.

Reply:
Ah, right, but isn't collect of `BitSet` guaranteed to be sorted also?